website/docs/feature-flag-tutorials/javascript/implementing-feature-flags-js.md

```diff
@@ -6,37 +6,36 @@ slug: /feature-flag-tutorials/javascript
 
 import VideoContent from '@site/src/components/VideoContent.jsx';
 
-Hello! In this tutorial we'll show you how to add feature flags to a plain JavaScript app, using [Unleash](https://www.getunleash.io/) and the official [Unleash Browser SDK](/reference/sdks/javascript-browser). With Unleash, an open-source feature flag service, you can add feature flags to your application and release new features faster.
+In this tutorial we'll show you how to add feature flags to a plain JavaScript app, using [Unleash](https://www.getunleash.io/) and the official [Unleash Browser SDK](/reference/sdks/javascript-browser). With Unleash, an open-source feature flag service, you can add feature flags to your application and release new features faster.
 
-In this tutorial, we'll make a basic website all about... Corgis! We'll use the [dog.ceo API 🐶](https://dog.ceo/) with HTML, CSS, JS to retrive some images of Queen Elisabeth's favourite dogs. We'll use feature flags to decide whether to show some corgi fun facts alongside the images.
+In this tutorial, we'll make a basic website Corgis! We'll use the [dog.ceo API 🐶](https://dog.ceo/) with HTML, CSS, JS to retrieve some images of Queen Elizabeth's favorite dogs. We'll use feature flags to decide whether to show some Corgi fun facts alongside the images.
 
 ## Prerequisites
 
 For this tutorial, you'll need the following:
-
 -   Git
 -   Docker and Docker Compose
--   A modern browser
+-   A browser or IDE; you can also use [CodeSandbox](https://codesandbox.io/docs/learn)
 
 ![architecture diagram for our implementation](./diagram.png)
 
-The Unleash Server is a **Feature Flag Control Service**, which manages your feature flags and lets you retrieve flag data. Unleash has a UI for creating and managing projects and feature flags. You can perform the same actions straight from your CLI or server-side app using the [Unleash API](/reference/api/unleash).
+The Unleash Server is a **Feature Flag Control Service**, which manages your feature flags and lets you retrieve flag data. Unleash has a UI for creating and managing projects and feature flags. In this example, we'll set up our script to make a request to Unleash using the Unleash SDK and show/hide a DOM element based on the flag data we receive. 
 
-## Install a local feature flag provider
+## Install Unleash
```

Comment:
For me, this tutorial is so Unleash-specific that I would prefer to just say Install Unleash instead of keeping it generic